### PR TITLE
[RFR] Custom messages for ezticket 401 errors

### PR DESCRIPTION
**Pull request by djhi**
[Trello card](https://trello.com/c/SY5gvbHw/15-0-5-bibcnrs-modifier-le-libelle-des-messages-d-erreur-unauthorized)

401 Page

### CODE REVIEW

File: lib/controller/ezticket/index.js [hunk 1]
```diff
@@ -25,6 +25,21 @@ app.use(route.get('/login', function* () {
...
+// Custom 401 handling to override default jwt message
+app.use(function *(next){
+    try {
+        yield next;
+    } catch (err) {
+        if (401 !== err.status) throw err;
+
+        this.status = 401;
+        this.body = `
+Vous ne pouvez accéder à cette ressource car vous interrogez un domaine autre que votre domaine autorisé. Veuillez relancer votre requête en sélectionnant votre domaine.\n\n
```
Comment (ThieryMichel) [inline lib/controller/ezticket/index.js]: we need a nice html error page. see https://trello.com/c/SY5gvbHw/15-0-5-bibcnrs-modifier-le-libelle-des-messages-d-erreur-unauthorized

Comment (djhi) [inline lib/controller/ezticket/index.js]: Any template available ?

Comment (ThieryMichel) [inline lib/controller/ezticket/index.js]: I used koa-send to serve static file

Comment (djhi) [inline lib/controller/ezticket/index.js]: No I meant an html template for design :)

Comment (djhi) [inline lib/controller/ezticket/index.js]: Found one with login.html

**Comment by ThieryMichel** [comment]
:+1:

**Pull request merged**